showcase/main_test.go

```diff
@@ -0,0 +1,49 @@
+// Copyright 2020 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package showcase_integration
+
+import (
+	"context"
+	"flag"
+	"log"
+	"os"
+	"testing"
+
+	showcase "github.com/googleapis/gapic-showcase/client"
+	"google.golang.org/api/option"
+	"google.golang.org/grpc"
+)
+
+func TestMain(m *testing.M) {
```

Comment:
Looks like `echo` and `identity` only have one test each. Why have a `TestMain` and not create the clients in the tests?

Reply:
When you say "one test each" do you mean one test _file_? Or one test _function_? If the latter, `echo` has many test functions, run in parallel, in `echo_test.go`. 

`identity` has one only test function, I concede, but I didn't want to duplicate the set up code. 

Happy to change things, but can you clarify?

Reply:
Makes sense. I thought it was one function for some reason, but it's clearly not.

Reply:
Thanks for the review!